zfs/replicate/__init__.py

```diff
@@ -1 +1,2 @@
+# -*- coding: utf-8 -*-
```

Comment:
Aren't these superfluous for Py3 code?

Reply:
Probably, but I probably went overboard with the pre-commit hooks.  We can remove them in another pull request if they become problematic or we see better evidence to drop them.